v8/client.go

```diff
@@ -17,7 +18,61 @@ type ClientMock struct {
 
 	mock.Mock
 	redis.Cmdable
-	client *redis.Client
+	client redis.UniversalClient
+	ctx    context.Context
```

Comment:
How does this get set if the test intends to mock `Context()`? Perhaps call it `Ctx`?